lib/messageUtils.ts

```diff
@@ -1,26 +1,70 @@
 import { getLookerSubmissionTimestampDateFilter } from "./lookerUtils";
 
-export function getDisplayNameForTemplate(template: string): string {
-  const displayNames: any = {
-    aboutwelcome: "About:Welcome Page (1st screen)",
-    defaultaboutwelcome: "Default About:Welcome Message (1st screen)",
-    feature_callout: "Feature Callout (1st screen)",
-    infobar: "InfoBar",
-    milestone_message: "Milestone Messages",
-    multi: "1st of Multiple Messages",
-    pb_newtab: "Private Browsing New Tab",
-    protections_panel: "Protections Dropdown Panel",
-    toast_notification: "Toast Notification",
-    toolbar_badge: "Toolbar Badge",
-    spotlight: "Spotlight Modal Dialog",
-    update_action: "Moments Page",
+export type SurfaceData = {
+  surface: string;
+  tagColorIndex: number;
+  docs?: string;
+};
+
```

Comment:
So since we're kind of already manually defining each surface, we can directly define `tagColor` here, and then we just have to make sure we're accessing it as part of SurfaceData in columns.tsx, for ex:

```
aboutwelcome: {
    surface: "About:Welcome Page (1st screen)",
    tagColor: "bg-red-400",
  },
```